instrumentation/log4j/log4j-2.13.2/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/log4j/v2_13_2/Log4j2InstrumentationModule.java

```diff
@@ -39,8 +39,7 @@ public Log4j2InstrumentationModule() {
 
   @Override
   public List<TypeInstrumentation> typeInstrumentations() {
-    // have to return at least 1 type instrumentation so that helpers get injected
-    return singletonList(new EmptyTypeInstrumentation());
+    return Arrays.asList(new BugFixingInstrumentation(), new EmptyTypeInstrumentation());
```

Comment:
Can you remove `EmptyTypeInstrumentation`? It's not needed anymore, since there's an actual type instrumentation in this module.

Reply:
The bugfix only applies to one of the codepaths in log4j, if it's using the other one it won't ever be applied so still need it.